NUEntity.js

```diff
@@ -9,7 +9,7 @@ export default class NUEntity extends NUObject {
     static attributeDescriptors = {
         creationDate: new NUAttribute({
             localName: 'creationDate',
-            attributeType: NUAttribute.ATTR_TYPE_NUMBER,
+            attributeType: NUAttribute.ATTR_TYPE_STRING,
```

Comment:
this should be a readonly attribute in the server. We need to ask server side to push it as readonly. Let's treat this as readonly in the UI

Reply:
ok